src/pages/index.astro

```diff
@@ -10,6 +10,9 @@ changeLanguage("en");
 
 const title = t("site.title");
 const description = t("site.description");
+const pluginsList = data.plugins.filter(function(plugin){
```

Comment:
No point displaying all the old ones. I've added text to send people to the full list if they want to help with the updating